pkg/cmd/version/version.go

```diff
@@ -1,34 +1,58 @@
 package version
 
 import (
+	"fmt"
+	"io"
+	"os"
+
 	"github.com/jenkins-x/jx-helpers/v3/pkg/cobras/helper"
-	"github.com/jenkins-x/jx-helpers/v3/pkg/termcolor"
-	"github.com/jenkins-x/jx-logging/v3/pkg/log"
 	"github.com/spf13/cobra"
 )
 
 // Build information. Populated at build-time.
-var (
+type BuildInfo struct {
 	Version   string
 	Revision  string
 	Branch    string
 	BuildUser string
 	BuildDate string
 	GoVersion string
-)
+}
+
+//Instance of the build info
+var buildInfo BuildInfo
```

Comment:
any reason to have a global variable?

Reply:
nvm, I see it.

Reply:
Isn't this the way to set using the makefile?